src/resolvers/index.ts

```diff
@@ -99,6 +101,13 @@ export function executeMutation<Args, ReturnType>(
     context: IContext
   ): Promise<ReturnType> {
     context.dbClient = writeClient();
+    Object.entries(args).forEach(([name, value]) => {
```

Comment:
This ignores nested fields for now, right?
Ex. `{ input: { id: 1, tags: ["dr", "dc"] } }`, we don't check the bulk size limit for `tags`

Reply:
yes right now we don't... thinking if we should 💭 

how bad can a savedItem with 10k tags, and 5 such savedItem nested in a mutation input would cost us.

we anyway have to return the all the tags we just added 1tag to a savedItem with 10k another tags.

Reply:
yeah it ignores nested fields... let me check our schema to see how much that would affect us